<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <arg name="angle" default="0.5235987756" />

  <node name="link_camera_1" pkg="tf" type="static_transform_publisher" args="0 0.08 0 $(arg angle) 0 0 camera_link camera_2_link 10"/>

  <node name="link_camera_2" pkg="tf" type="static_transform_publisher" args="0 -0.08 0 -$(arg angle) 0 0 camera_link camera_1_link 10"/>

<node name="link_laser" pkg="tf" type="static_transform_publisher" args="0.05 0 0.23 0 0 0 camera_link laser 10"/>

  <node name="link_imu" pkg="tf" type="static_transform_publisher" args="-0.02 0 0 0 0 0 camera_link imu 10"/>

  <node name="link_sensors" pkg="tf" type="static_transform_publisher" args="0.04 0 0.2 0 0 0 base_link camera_link 10"/>

  <include file="$(find featureless_and_high_dyamic_navigation)/launch/include/multiple_astra_1.launch.xml"/>
  <include file="$(find featureless_and_high_dyamic_navigation)/launch/include/multiple_astra_2.launch.xml"/>

  <node name="rplidarNode"          pkg="rplidar_ros"  type="rplidarNode" output="screen">
    <param name="serial_port"         type="string" value="/dev/ttyUSB1"/>  
    <param name="serial_baudrate"     type="int"    value="115200"/>
    <param name="frame_id"            type="string" value="laser"/>
    <param name="inverted"            type="bool"   value="false"/>
    <param name="angle_compensate"    type="bool"   value="true"/>
    <param name="frame_id"            type="string" value="laser"/>
  </node>

  <node name="imu" pkg="featureless_and_high_dyamic_navigation" type="brick_callback.py" />

</launch>
